payloads/start.go

```diff
@@ -127,6 +127,9 @@ type RequestedResource struct {
 	// Value specifies the integer value associated with that resource.
 	Value int `yaml:"value"`
 
+	// ValueString is an optional string format value instead of integer
+	ValueString string `yaml:"value_string,omitempty"`
```

Comment:
@tpepper does it make sense to add a satisfies method to things like strings that are not directly comparable. basically create a new type here with an associated method?